### PR TITLE
Add filterable function for payment details IP addresses

### PR DESCRIPTION
**Pull request by davidsherlock**

### CODE REVIEW

File: includes/misc-functions.php [hunk 1]
```diff
@@ -950,3 +950,21 @@ function edd_can_view_receipt( $payment_key = '' ) {
...
+
+
+/**
+ * Given a IP address, generate a link to IP address provider (ipinfo.io)
+ *
+ * @since  2.8.15
+ * @param  string	$ip_address The Payment IP Address
+ * @return string	A link to the IP details provider
+ */
+function edd_payment_get_ip_address_url( $ip_address ) {
```
Comment (pippinsplugins) [inline includes/misc-functions.php]: Let's include the payment ID as a parameter here.

File: includes/misc-functions.php [hunk 2]
```diff
@@ -950,3 +950,21 @@ function edd_can_view_receipt( $payment_key = '' ) {
...
+
+	$base_url = 'https://ipinfo.io/';
+	$provider_url = '<a href="' . esc_url( $base_url ) . esc_attr( $ip_address ) . '" target="_blank">' . esc_attr ( $ip_address ) . '</a>';
+
+	return apply_filters( 'edd_payment_get_ip_address_url', $provider_url );
```
Comment (pippinsplugins) [inline includes/misc-functions.php]: This filter should include the original IP and the payment ID as a parameter.

**Comment by cklosowski** [comment]
@davidsherlock Still seeing references to `$payment->get_ip()` instead of just `$payment->ip`. Can we get those resolved, otherwise this looks good.

**Comment by davidsherlock** [comment]
Fixed @cklosowski

**Pull request merged**